docs/source/r-and-r-essentials.rst

```diff
@@ -33,10 +33,50 @@ Now you can easily share your new meta-package with friends and colleagues by up
   anaconda login
   anaconda upload path/to/custom-r-bundle-0.1.0-0.tar.bz2
 
-Your friends and colleagues now have access to your Custom-R-Bundle from any computer with the command:
+Your friends and colleagues now have access to your Custom-R-Bundle from any computer with the command::
 
-``conda install -c <your anaconda.org username> custom-r-bundle``
+  conda install -c <your anaconda.org username> custom-r-bundle
 
 For more information, see Christine Doig's blog post `Jupyter and conda for R <https://www.continuum.io/blog/developer/jupyter-and-conda-r>`_.
 
+Microsoft R Open
+================
+
+You can install `Microsoft R Open (MRO) <https://mran.revolutionanalytics.com/download/mro-for-mrs/>`_ with conda on 64-bit Windows, 64-bit OS X, and 64-bit Linux, from the Anaconda.org channel "mro".
+
+There are two ways to install MRO and other R packages. The first is to add the "mro" channel to your :doc:`.condarc configuration file<config>` above the "r" channel, and then use ``conda install r`` and use conda to install other packages. The second is to specify the "mro" channel on the command line each time you use the conda install command: ``conda install -c mro r``
+
+In addition to installing R packages from conda channels, it is possible to install R packages from the Comprehensive R Archive Network (CRAN) or the Microsoft R Application Network (MRAN). These R packages will install into the currently active conda environment. MRO includes the `checkpoint package <https://github.com/RevolutionAnalytics/checkpoint/>`_, which installs from MRAN and is designed to offer enhanced `reproducibility <https://mran.revolutionanalytics.com/documents/rro/reproducibility/>`_.
+
+To use MRO or R packages, activate the conda environment where they are installed to set your environment variables properly. Executing the programs at the pathname in that environment without activating the environment may result in errors.
+
+Each conda environment may have packages installed from the channel "r" or the channel "mro", but no conda environment should contain packages from both channels, which may result in errors. If this occurs, you may use ``conda remove`` to remove the packages that were installed incorrectly, or create a new environment and install the correct packages there.
+
+Installing the Intel Math Kernel Library (MKL) with Microsoft R Open
+====================================================================
+
+The Intel Math Kernel Library (MKL) extensions are also available for MRO on Windows and Linux, while the Accelerate library is used instead on OS X. Just `download the MKL package for your platform <https://mran.revolutionanalytics.com/download/>`_ and install it according to the instructions.
+
+NOTE: Installing the MKL package constitutes implicit agreement to the `MKL license <https://mran.revolutionanalytics.com/assets/text/mkl-eula.txt>`_.
```

Comment:
By using our install process, we assume that you have already agreed to the MKL license.  Proceeding further is an implicit agreement to said license.